autopts/wid/gatt.py

```diff
@@ -1686,7 +1686,13 @@ def hdl_wid_122(_: WIDParams):
     return '0000'
 
 
-def hdl_wid_130(_: WIDParams):
+def hdl_wid_130(params: WIDParams):
+    btp.gap_unpair()
+
+    # This is needed until ES-27410 is incorporated
+    if params.test_case_name in ['GATT/SR/GAI/BV-01-C', 'GATT/SR/GAN/BV-01-C']:
+       btp.gap_set_bondable_off();
```

Comment:
Should `ES-27410` be added to the errata list?

Reply:
I'll do it when I get initial feedback on it